src/ffigen/ffigen.py

```diff
@@ -186,6 +235,15 @@ def need_cwrap(id):
             enum_value_value = enum_value.get("init")
             print(f"const _{enum_name}_{enum_value_name} = {enum_value_value};")
 elif mode == "cwrap":
+    # Generate C accessors for simple types
```

Comment:
Why? If we are importing multiple headers, are they all going to have these definitions? Also, shouldn't these be inline, in a header, likely in static_h.h, or at least a parallel one?